src/android/ConnectPlugin.java

```diff
@@ -334,6 +333,45 @@ public boolean execute(String action, JSONArray args, final CallbackContext call
             logger.logEvent(AppEventsConstants.EVENT_NAME_COMPLETED_REGISTRATION, params);
             callbackContext.success();
             return true;
+        } else if (action.equals("logCompletedTutorial")) {
+            logger.logEvent(AppEventsConstants.EVENT_NAME_COMPLETED_TUTORIAL);
+            callbackContext.success();
+            return true;
+        } else if (action.equals("logInitiatedCheckout")) {
+            if (args.length() != 2) {
+                callbackContext.error("Invalid arguments");
+                return true;
+            }
+            JSONObject parameters;
+            String contentData;
```

Comment:
Missing the initializers here (for all these variables)

Reply:
Added them in now!